base58/base58.go

```diff
@@ -21,8 +19,8 @@ func Decode(b string) []byte {
 	j := big.NewInt(1)
 
 	for i := len(b) - 1; i >= 0; i-- {
-		tmp := strings.IndexAny(alphabet, string(b[i]))
-		if tmp == -1 {
+		tmp := b58[b[i]]
```

Comment:
Also, since the table is being autogenerated now, would it make sense to just autogenerate the b58 arrays with `big.Ints` instead of doing a lookup and creating a new big integer each time?